### PR TITLE
EPMFARMATS-12727

### PR DESCRIPTION
**Pull request by overdd**

### CODE REVIEW

File: EPMFARMATS-12727-protractor.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    const mainSearchBox = element(by.id('supHomeAndLandingPageSearchBox'));
+    const feedbackYesButton = element(by.name('buttonYes'));
+    const feedbackTextArea = element(by.name('feedbackText'));
+    const feedbackSendCommentButton = element(by.css('.ocButtonSendComment'));
+    const feedbackThankYouMessage = element(by.xpath("//*[text()='Thank you for your feedback!']"));
+
+    beforeAll(function () {
+        browser.waitForAngularEnabled(true);
+        browser.get('https://support.microsoft.com/en-us/office');
+        browser.manage().window().maximize();
```
Comment (SiarheiRylach) [inline EPMFARMATS-12727-protractor.js:14]: advice: better to use particular window size.

File: EPMFARMATS-12727-protractor.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    });
+
+
+    it('should get Thank You message after submitting a feedback', () => {
+        mainSearchBox.click().then(() => {
```
Comment (SiarheiRylach) [inline EPMFARMATS-12727-protractor.js:19]: for future: I would recommend using async/await or promise chaining for async functions. I wouldn't hope on control flow.

File: conf.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+
+exports.config = {
+    framework: 'jasmine',
+    seleniumAddress: 'http://localhost:4444/wd/hub',
```
Comment (SiarheiRylach) [inline conf.js:4]: directConnect is better option in your case

File: EPMFARMATS-12727-protractor.js (hunk 1 above)
Comment (overdd) [reply EPMFARMATS-12727-protractor.js]: fixed

File: EPMFARMATS-12727-protractor.js (hunk 2 above)
Comment (overdd) [reply EPMFARMATS-12727-protractor.js]: fixed

File: conf.js (hunk 3 above)
Comment (overdd) [reply conf.js]: Fixed

**Review by SiarheiRylach — APPROVED** [review]

**Pull request merged**